covfee/client/tasks/continuous_annotation/slice.ts

```diff
@@ -1,14 +1,12 @@
 import { createSlice } from "@reduxjs/toolkit"
 
 export interface State {
-  active_annotation?: string
-  active_participant?: string
+  selectedAnnotationIndex?: number
   mediaPaused: boolean
 }
 
 export const initialState: State = {
-  active_annotation: null,
-  active_participant: null,
+  selectedAnnotationIndex: -1,
```

Comment:
Why not have the initial state be `null` too? It's a bit more explicit than using -1

Reply:
Good point. Still getting used to Javascript :)